test/utils/utils.test.js

```diff
@@ -400,4 +400,13 @@ describe('Utils', () => {
       expect(document.title).to.equal(expected);
     });
   });
+
+  describe('seotech', async () => {
+    beforeEach(async () => {
+      document.head.innerHTML = await readFile({ path: './mocks/head-seotech-video.html' });
+    });
+    it('should import and call default with falsey value', async () => {
+      await utils.loadArea();
```

Comment:
Should there be an expect statement in here?

Reply:
[cde384d](https://github.com/adobecom/milo/pull/1065/commits/cde384da3990ca7678cb02703f6b24819b72dccb)